nextcord/components.py

```diff
@@ -230,43 +288,212 @@ class SelectMenu(Component):
     options: List[:class:`SelectOption`]
         A list of options that can be selected in this menu.
     disabled: :class:`bool`
-        Whether the select is disabled or not.
+        Whether the select is disabled or not. Defaults to ``False``.
+    channel_types: List[:class:`ChannelType`]
+        A list of channel types that are allowed to be chosen in this select menu.
```

Comment:
It seems like channel_types only needs to be mentioned in the subclass and not here